docs/onboarding/VPN.md

```diff
@@ -1,10 +1,21 @@
-## General documenation about connecting to Boulder office's VPN
+## General documenation about connecting EF VPNs
 
-**Client configuration:**
+We use [wireguard](https://www.wireguard.com/) as our VPN of choice. Our wireguard setup allows you to get access to the EF Berlin
+office as well as the EF Boulder office. If you need a third, private and secure VPN just for your teammates to access servers you host
+on our infra then please reach out to the DevOps team.
 
+The below guide assumes that you have `wireguard` installed (Run ` wg --help` to check).
```

Comment:
Maybe add that if you use mac, you can also just use the wireguard from the appstore if you prefer to have a gui.

Reply:
added